packages/react-storage/src/components/StorageBrowser/Views/utils.ts

```diff
@@ -27,3 +27,10 @@ export const listViewHelpers = ({
     range: [start, end],
   };
 };
+
+export const getFileExtension = (fileName: string): string => {
+  const indexOfDot = fileName.lastIndexOf('.');
+
+  // Return '-' when no file extension found from file name, similar to S3 console
```

Comment:
Prefer avoidance of comments pertaining to external code. It creates fragility, what if the other implementation changes?